include/dfu/dfu_target_suit.h

```diff
@@ -73,21 +74,23 @@ int dfu_target_suit_write(const void *const buf, size_t len);
 
  * @param[in] successful Indicate whether the firmware was successfully recived.
  *
- * @return 0 on success, negative errno otherwise.
+ * @retval 0 on success, negative errno otherwise.
  */
 int dfu_target_suit_done(bool successful);
 
 /**
  * @brief Schedule update and reset the device.
```

Comment:
@brief Schedule update and **reset** the device.

Shouldn't it be:

@brief Schedule an update

?